essentials.sh

```diff
@@ -33,6 +49,14 @@ text_in_yellow_color=$(tput setaf 3)
 
 reset=$(tput sgr0)
 
+# Images types
+
+IMAGE_FOR_RPI=1
```

Comment:
How about "PLAIN_IMAGE"

Reply:
I wanted all the params to start with the `IMAGE_` prefix.